### DISCUSSION TITLE
Update single dependency across all projects in a Poetry monorepo

### DISCUSSION THREAD

**Discussion by pepastach** [renovatebot/renovate#19449] [Request Help] [answered]
Hi everyone,

I've been testing Renovate for a couple of days but I can't find a solution to this particular problem:

* I have a python monorepo with many Poetry projects. Many of them depend on one another.
* When renovate creates a MR for a single dependency update (let's say package `networkx`), it does so in the one project (`project_x`), that has this dependency. But there are other Poetry projects that depend on `project_x` and their poetry.lock files need to be updated as well.

How do I tell Renovate that once it bumps a single dependency, it needs to check if other projects depend on the modified project?

Thank you!

**Comment by rarkins** [comment, marked as answer]
Unfortunately this type of cross-project awareness is not yet supported for Poetry. Some other ecosystems like Node are better than Python/Poetry and handle cross-project awareness themselves using "workspaces" concepts so Renovate doesn't need to be so aware of which projects depends on which others. Do you have or can you create a public example of this cross-project dependency?

**Reply by pepastach** [reply]
Thank you. I can prepare a minimal example of such rpository. Is this feature something, you'd consider implementing to Renovate?

**Reply by rarkins** [reply]
Yes, we have similar for other managers
